lib/admin/WP_Auth0_Admin_Basic.php

```diff
@@ -73,7 +73,7 @@ public function init() {
 				'function' => 'render_client_signing_algorithm',
 			),
 			array(
-				'name'     => __( 'Cache Time (in minutes)', 'wp-auth0' ),
+				'name'     => __( 'JWKS Cache Time (in minutes)', 'wp-auth0' ),
```

Comment:
Should this be a lower-case S?

Reply:
And also are they going to know what JWK is?

Reply:
Not officially:
https://auth0.com/docs/jwks

> And also are they going to know what JWK is?

If they don't, they shouldn't be screwing around with it 😃